### ISSUE TITLE
Cartputer runs out of power but doesn't shut down

### ISSUE THREAD

**Issue by Forecaster** [feldim2425/OC-Minecarts#12]
http://imgur.com/bwv7PJk

ocminecart 1.5-oc1.6
Minecraft 1.7.10
OpenComputers 1.6.0.907-dev

**Comment by feldim2425** [comment]
Hmm. I don't see a big differece between my code and the code from the OC Drone. Maybe it's a OC issue. Not sure.

**Comment by feldim2425** [comment]
Sorry but I cannot reproduce it.

I used the same OC and OC-Minecart version, but the Cart turned off the Engine and the internal computer. (as you can see in the screenshot)

![bildschirmfoto vom 2016-07-07 23-52-52](https://cloud.githubusercontent.com/assets/6705744/16671135/082a2c2c-449e-11e6-88da-0ce2eff28ddc.png)

**Comment by Forecaster** [comment]
Hm... um, I was in the nether when this happened? (I was pumping lava)

Doubt that'd matter, if not that it'd be one of the other mods in my modpack.

You can find my modlist here: http://towerofawesome.org/youtube/modlist

As well as a link to download my modpack, although my system isn't working quite right, so not all mods are included for some reason. I've yet to fix that.

When I have time I'll do some more testing with this and see if I can narrow down the problem mods.

**Comment by feldim2425** [comment]
Can you try if the bug also happens with drones and/or robots? 

Or reading the amount of energy in lua. Just start the "lua" program and use computer.getEnergy()

**Comment by feldim2425** [comment]
Can you try if the bug also happens with drones and/or robots? Because

**Comment by feldim2425** [comment]
Can you try if the bug also happens with drones and/or robots? Because

**Comment by Forecaster** [comment]
I figured it out. It's a server-client sync issue. It doesn't happen in SP, but happens on my dedicated server, which is where I first tried making a cart for testing.

The energy reported by Lua is indeed correct, but the GUI is displaying the wrong amount and quickly goes negative. But in a singleplayer world it works correctly.

And regular OC robots and drones work correctly on the server, it's just the carts.
